### PR TITLE
styling the game board screen

### PR DESCRIPTION
**Pull request by mche791**
score UI
 - added design for the score, line, and level count
 - changed the labels to properly match each respective label of score, line, and level
 
game board
 - slight changes to the main gameboard

next piece UI
 - made same design for the next piece UI
 
hold piece UI
 - made same design for the hold piece UI
 
gameboard UI
 - small adjustments to gameboard UI

Title + esc button
 - added style to escape button + title

 - find proper background to game
 - currently pause button has no functionality

### CODE REVIEW

File: src/main/java/com/stackedsuccess/controllers/GameBoardController.java [hunk 1]
```diff
@@ -318,4 +325,12 @@ public void gameOver() throws IOException {
...
         System.exit(0);
     }
+
+    /**
+     * Method for when pressing the pause button, this will pause the current game instance
+     *
+     * @throws IOException
+     */
+    @FXML
+    public void onEscapeButton(ActionEvent event) throws IOException {}
```
Comment (axu732) [inline src/main/java/com/stackedsuccess/controllers/GameBoardController.java:335]: Please delete this method. OnKeyPressed links to the GameInstance method handleInput() which already deals with pressing the ESC button to pause.

**Review by axu732 — CHANGES_REQUESTED** [review]
Please review the changes needed in gameboardcontroller, then this pr can be accepted.

**Comment by mche791** [comment]
> Please review the changes needed in gameboardcontroller, then this pr can be accepted.

Method should be deleted now 👍

**Review by axu732 — APPROVED** [review]

**Comment by axu732** [comment]
LGTM

**Pull request merged**